src/metabase/driver/postgres.clj

```diff
@@ -196,7 +196,7 @@
        :unix-timestamp->timestamp         unix-timestamp->timestamp
        :date                              date
        :date-interval                     date-interval
-       :set-timezone-sql                  "SET LOCAL timezone TO ?;"
+       :set-timezone-sql                  "UPDATE pg_settings SET setting = ? WHERE name ILIKE 'timezone';"
```

Comment:
Yup, UPDATE on `pg_settings` [only affects the value used by the current session](http://www.postgresql.org/docs/9.1/static/view-pg-settings.html)